modules/importer/src/server/mod.rs

```diff
@@ -151,6 +118,11 @@ async fn import(
     importer: Importer,
     service: ImporterService,
 ) -> Result<(), Error> {
+    let locked = Heart::beat(&importer, &runner.db)
+        .await
+        .map_err(|_| Error::MidAirCollision)?;
```

Comment:
Wouldn't that lead to a failed importer run? Reported in the UI?

Reply:
That means that **any** DB gets turned into a "mid air collision"?

Reply:
What would happen if no row was updated? Would that return an error too?

Reply:
Yes. I confirmed that. I was happy to discover that calling `::Entity::update` returns an error if no row is updated, unlike `update_many`.

Reply:
Please ignore that question above then. I think it would be good to at least document that in our code. As to me, it's not obvious.